api/v1alpha1/tls_types.go

```diff
@@ -9,6 +9,9 @@ import (
 	gwapiv1 "sigs.k8s.io/gateway-api/apis/v1"
 )
 
+// TLSOCSPKey is the key for the OCSP stapled response in a Secret.
+const TLSOCSPKey = "tls.ocsp-staple"
```

Comment:
can we use tls.ocsp here as suggested in https://github.com/envoyproxy/gateway/issues/3826#issuecomment-3109912989